### ISSUE TITLE
"encoding type" button is not labelled

### ISSUE THREAD

**Issue by SteveChab** [cmu-delphi/www-covidcast#239]
The "encoding type" button is not labelled. The group is labelled as "encoding type," which is somewhat vague. But the button itself is not labelled. My screen reader (Voiceover) simply reads, "Toggle button" and doesn't say what it's toggling.

![image](https://user-images.githubusercontent.com/275785/88084377-514d5200-cb52-11ea-864a-6a51ad23673f.png)
